### ISSUE TITLE
Synthese : Erreur 500 quand champ taxon remplis mais aucun taxon sélectionné

### ISSUE THREAD

**Issue by amandine-sahl** [PnX-SI/GeoNature#824]
Lorsque une (ou des) lettre est saisie dans le champ taxon, sans qu'un taxon ne soit selectionné pour autant, le POST envoie le paramètre cd_nom=lettre tapée ce qui provoque une erreur 500.
Il faudrait mettre un test coté client/serveur pour éviter ce problème.

![image](https://user-images.githubusercontent.com/3088650/70980556-cab02e00-20b3-11ea-9eed-ae6c6afc03e1.png)

**Comment by camillemonchicourt** [comment]
On voulait pouvoir chercher par cd_nom aussi si besoin. 
C'est plus possible du coup ?

**Comment by amandine-sahl** [comment]
Effectivement, est ce qu'il ne faudrait pas un champ dans le formulaire avancé pour que ce soit moins confusant?
Sinon autre solution tester si la valeur du champ cd_nom est de type integer.

**Comment by camillemonchicourt** [comment]
C'était vraiment simple d'avoir un seul champs où je recherche en latin, en français, et si besoin (fonction un peu cachée), je saisis directement le CD_NOM. 
Pas trop pour faire un autre champs.

**Comment by amandine-sahl** [comment]
ok on peut partir sur un test : type==integer dans ce cas là

**Comment by camillemonchicourt** [comment]
OK !

**Comment by TheoLechemia** [comment]
Ça a été corrigé dans la Synthèse, mais il y a toujours le problème dans la recherche Occtax.

**Comment by jacquesfize** [comment]
Résolu au mois dans la 2.15.x
